NorthstarDLL/util/printmaps.cpp

```diff
@@ -142,6 +143,27 @@ int, __fastcall, (const char const* cmdname, const char const* partial, char com
 	return numMaps;
 }
 
+ADD_SQFUNC(
```

Comment:
Why do you expose this to Squirrel VM?

Reply:
When changing a map either using the `map` command or `GameRules_ChangeMap` you need to enter a map the game can load, otherwise you engine error. For example FSU2 [checks](https://github.com/F1F7Y/ServerUtilities/blob/main/mods/Fifty.ServerVote/mod/scripts/vscripts/fsv_localization.nut#L81-L124) against a predefined array to make sure we dont try to change to an invalid map. The list being hardcoded obviously is a problem especially for custom maps.

Reply:
@Alystrasz can this be resolved?

Reply:
Yep, I thought @F1F7Y was gonna close it :)